### DISCUSSION TITLE
Need to reinstall c/c++ every time reconnecting to the server

### DISCUSSION THREAD

**Discussion by hpphappy** [microsoft/vscode-cpptools#9596] [Q&A]
I have the extension of Remote - SSH installed, and I use it to connect to a remote server. I found that I need to reinstall the extension "C/C++" (id: ms-vscode.cpptools) in every reconnection, while "C/C++ Extension Pack" (id: ms-vscode.cpptools-extension-pack) only requires the installation once.  
Does anyone know how I could keep "C/C++" installed? Thank you.

The information in "Help/About":
Version: 1.69.1 (user setup)
Commit: b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a
Date: 2022-07-12T08:21:24.514Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.19042

**Comment by Colengms** [comment]
Hi @hpphappy .  That would seem to be a VS Code issue, as the C/C++ extension does not control its own installation or removal.  You might consider opening an issue in [the VS Code repo](https://github.com/microsoft/vscode/issues).

**Reply by sean-mcmanus** [reply]
Also, the C++ extension installs on the remote machine so if you're connecting to different remote machines then every one would need a new install. Otherwise, we've tested with the Remote SSH extension and haven't seen this issue.
